LayoutTests/imported/w3c/web-platform-tests/css/css-text/letter-spacing/letter-spacing-animating-letter-spacing.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>letter-spacing animates when animating letter-spacing</title>
+<link rel="author" title="Sam Weinig" href="mailto:sam@webkit.org">
+<link rel='help' href='https://www.w3.org/TR/css-text-3/#letter-spacing'>
+<link rel='match' href='reference/letter-spacing-animating-letter-spacing-ref.html'>
+</head>
+<body>
+<div>The letters below should be separated by 20 pixels.</div> 
+<div id="target">ABCD</div>
+
+<script>
+  const target = document.querySelector('#target');
+
+  let animation = target.animate(
+    [
+      { letterSpacing: '0px' },
+      { letterSpacing: '40px' }
+    ],
+    {
+      duration: 40
+    }
+  );
```

Comment:
Replicating my earlier comments, you could write this as:

```suggestion
  const animation = target.animate({ letterSpacing: ['0px', '40px'] }, 40);
```